src/Extension/Resources/config/task/SprykRunTask.yaml

```diff
@@ -2,7 +2,7 @@
 id: "spryk:run"
 short_description: "Runs a Spryk build process."
 version: "0.1.0"
-command: "php %executable_command% %spryk% %targetModule% %dependentModule% %option%"
+command: "php %executable_command% %spryk% %organization% %coreLevel%"
```

Comment:
Can I as a developer provide additional options for the `spryk` tool?

Reply:
it possible like spryk:run --spryk=AddModuleDefault --option="--param1=Test" --option="--param2=Test" and etc